.github/workflows/docusaurus.yml

```diff
@@ -19,16 +19,37 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Check out repo
         uses: actions/checkout@v2
+
+      # Go is required to generate cli docs
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.work'
+          cache: true
+          cache-dependency-path: go.work
       # Node is required for npm
       - name: Set up Node
         uses: actions/setup-node@v2
         with:
           node-version: "18"
+
+      # build CLI docs
+      - name: Generate tracetest CLI reference docs
```

Comment:
if we were to have this, does it mean that the docs might be updated with CLI changes even if they are not released?

Reply:
this means that every time we release docs, it will include the cli docs that results from building the version for the commit being built

Reply:
However, these files are not committed in our repo in this way, making the need of creating a PR to update them into the code. If we want to see them locally on docusaurus we need to run `make docgen` on CLI and later run docusarus manually, right?

Thinking about this, does it makes sense to have the autogen file committed in our repo?